kernel/init.c

```diff
@@ -51,7 +51,7 @@ const char * const build_timestamp = BUILD_TIMESTAMP;
 /* boot time measurement items */
 
 #ifdef CONFIG_BOOT_TIME_MEASUREMENT
-u64_t __noinit __start_time_stamp; /* timestamp when kernel starts */
+u64_t __start_time_stamp; /* timestamp when kernel starts */
```

Comment:
This *has* to be noinit. The TSC is read right at the entry point before BSS is zeroed. If it is in BSS the stored value will be clobbered.